org/teams.yaml

```diff
@@ -515,43 +422,13 @@ teams:
       - louiscryan
       - nrjpoddar
     repos:
-      api: write
-      bots: write
-      client-go: write
-      cni: write
-      common-files: write
-      community: write
-      cri: write
-      enhancements: write
-      envoy: write
-      gogo-genproto: write
-      installer: write
-      istio: write
-      istio.io: write
-      operator: write
-      pkg: write
-      proxy: write
-      release-builder: write
-      test-infra: write
-      ztunnel: write
+      .default: write
   Triagers:
     description: Folks that perform issue & pr triage.
     members:
       - SkyfireFrancisZ
     repos:
-      api: none
-      bots: none
-      client-go: none
-      cni: none
-      cri: none
-      envoy: none
-      installer: none
-      istio: none
-      istio.io: none
-      operator: none
-      pkg: none
-      proxy: none
-      tools: none
+      .default: none # Not sure why this isn't "Triage"
```

Comment:
Honestly, I don't know why we have the Triagers section at this point. I think it was for some earlier testing, and Francis already has more permissions in another section.